raiden-ts/CHANGELOG.md

```diff
@@ -230,31 +271,36 @@
 - Fix logging disable not working properly.
 
 ### Removed
+
 - Remove Kovan network support.
 - Remove requirement for monitored token when calling getTokenInfo|getTokenBalance.
 
 ## [0.1.1] - 2019-10-07
+
 ### Added
+
 - Add RaidenChannels alias.
 - Add monitoring for transfers based on secret hash.
 
 ### Changed
-- Change transfer api return secret hash. 
+
+- Change transfer api return secret hash.
 
 ## [0.1] - 2019-08-21
+
 ### Added
+
 - Add token monitoring.
 - Add channel lifecycle integration (open/close/settle) with contracts.
 - Add channel deposit functionality.
-- Add channels$ to the public API.
+- Add channels\$ to the public API.
 - Add getTokenBalance and getTokenInfo to public API.
-- Add network and events$ to the public API.
+- Add network and events\$ to the public API.
 - Add account change and network change monitoring.
 - Add matrix sdk/transport integration.
 - Add protocol message implementation.
 
-
-[Unreleased]: https://github.com/raiden-network/light-client/compare/v0.9.0...HEAD
+[unreleased]: https://github.com/raiden-network/light-client/compare/v0.9.0...HEAD
```

Comment:
Why lowercase?

Reply:
Ah damn I forgot to disable auto-formatting for this file. There a lot of new new-lines etc. Need to revert that.

Reply:
:heavy_check_mark: